src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const discount7Days = 50;
+  const discount3Days = 20;
+  const longTermDays = 7;
+  const midTermDays = 3;
+
+  if (days >= longTermDays) {
+    return days * dailyRate - discount7Days;
```

Comment:
Create a variable for this logic and use it everywhere
```suggestion
  const midTermDays = 3;
  const basePrice = days * dailyRate;

  if (days >= longTermDays) {
    return basePrice - discount7Days;
```